### PR TITLE
Motion2d item movement

### PR DESCRIPTION
**Pull request by cdsupina**
Fixed item movement.

Removed unnecessary imports in systems.

### CODE REVIEW

File: src/systems/spaceship_movement.rs [hunk 1]
```diff
@@ -32,19 +37,62 @@ impl<'s> System<'s> for SpaceshipMovementSystem {
...
+    let x_pos = transform.translation().x - (hitbox.width / 2.0);
+    let y_pos = transform.translation().y - (hitbox.height / 2.0);
+
+    if x_pos < ARENA_MIN_X || x_pos > ARENA_MAX_X {
+        motion.velocity.x *= -1.0;
+    }
+
+    if y_pos < ARENA_MIN_Y || y_pos > ARENA_MAX_Y {
+        motion.velocity.y *= -1.0;
+    }
```
Comment (tigleym) [inline src/systems/spaceship_movement.rs:97]: It's odd how this change is showing up as part of your changed files.

File: src/systems/blast.rs [hunk 2]
```diff
@@ -37,11 +31,6 @@ impl<'s> System<'s> for BlastSystem {
                     .delete(blast_entity)
                     .expect("unable to delete entity");
             }
-
-            // update position based on blast velocity
-            blast_transform.prepend_translation_x(blast_motion2d.velocity.x * time.delta_seconds());
-            blast_transform
-                .prepend_translation_y((blast_motion2d.velocity.y) * time.delta_seconds());
```
Comment (tigleym) [inline src/systems/blast.rs:44]: I think these changes were part of the commit that fixed the blasts movement

**Review by tigleym — COMMENTED** [review]
This looks good! I'm a little confused about how the final file changes ended up including the work from previous commits (i.e: spaceship_movement and blasts). It looks like the main changes here were for the item system, entity, and constants.

File: src/systems/spaceship_movement.rs (hunk 1 above)
Comment (cdsupina) [reply src/systems/spaceship_movement.rs:97]: I wonder if it has something to do with changing the name of the repo.

Comment (cdsupina) [reply src/systems/spaceship_movement.rs:97]: I don't recall touching those files.

**Pull request merged**